internal/cephfs/clone.go

```diff
@@ -188,15 +195,14 @@ func createCloneFromSnapshot(ctx context.Context, parentVolOpt, volOptions *volu
 	return nil
 }
 
-// CloneStatus represents the subvolume clone status.
-type CloneStatus struct {
-	Status struct {
-		State string `json:"state"`
-	} `json:"status"`
-}
+func getCloneInfo(ctx context.Context, volOptions *volumeOptions, cr *util.Credentials, volID volumeID) (cephFSCloneState, error) {
+	type cloneStatus struct {
```

Comment:
What's the rationale for moving the type declaration into the function body? Wouldn't just lowercasing the name accomplish your goal?

Reply:
Yes, that too. But I'm preparing to just return go-ceph/cephfs/admin.CloneState.

Also, this was something I had laying around and could test the new **ci/centos/k8s-e2e-external-storage** job with :wink: